ietf/doc/views_search.py

```diff
@@ -446,6 +447,10 @@ def docs_for_ad(request, name):
         if blocked_docs:
             blocked_docs.sort(key=lambda d: min(p.time for p in d.blocking_positions if p.balloter==ad), reverse=True)
 
+        for d in blocked_docs:
+           if d.get_base_name() == 'charter-ietf-shmoo-01-04.txt':
+              print('Is in list')
```

Comment:
This looks like debugging code that snuck in

Reply:
Ooops.  I just removed it from my branch.

Reply:
And my error for letting it in.
See https://github.com/ietf-tools/datatracker/pull/3704